en/rocky/8/admin_guide/01-presentation.md

```diff
@@ -0,0 +1,299 @@
+# Introduction to the Linux operating system
+
+In this chapter you will learn about GNU/Linux distributions.
+
+****
+**Objectives** : In this chapter you will learn how to:
+
+:heavy_check_mark: Describe the features and possible architectures of an operating system;\
+:heavy_check_mark: Recount the history of UNIX and GNU/Linux;\
+:heavy_check_mark: Choose the right Linux distribution for your needs;\
+:heavy_check_mark: Explain the philosophy of free and opensource software;\
+:heavy_check_mark: Discover the usefulness of the SHELL.\
+
+
+To explain your goals, you should only use action verbs. Because goals need to be quantifiable, measurable and verifiable.
+At the end of the document, you can add questions directly related to these objectives to validate the understanding of the subject.
+
+:checkered_flag: **generalities**, **linux**, **distributions**
+
```

Comment:
The checkered_flag give this: 🏁
![image](https://user-images.githubusercontent.com/3603739/119549650-bb004300-bd97-11eb-98fa-1e8a83a1a4e7.png)
Only because I didn't find a better icon to represent tags.

If you dislike, we can remove the flag and the tags. NP.

Reply:
Let's leave it alone. I think this is a problem with my brain! Ha!